variables.tf

```diff
@@ -1,9 +1,3 @@
-variable "tamr_service_account" {
```

Comment:
My preference would be to make this a required field by removing the default value and have.

It provides more flexibility for the user of this module to either create a new service account or use an existing one, vs this change which always requires a new service account.